File: DAY-4/Print_nto1.cpp
#include<bits/stdc++.h>
using namespace std;

void toPrint(int n){
	if(n==0){
		return;
	}

	cout<<n<<endl;
	toPrint(n-1);
}

int main(){
	int n;
	cin>>n;
	toPrint(n);
}

File: DAY-4/Print_n_to_1.java
package com.CherishDSA;

public class Print_n_to_1 {
    public static void toPrint(int n){
        if(n==0){
            return;
        }
        System.out.println(n);
        toPrint(n-1);
    }
    public static void main(String[] args) {
        toPrint(5);
    }
}
